### PR TITLE
[WIP] Create degrees_to_rads

### PR DESCRIPTION
**Pull request by fluctix**
## Description 

Added a function which converts degrees to radians.

For a [GCI Task](https://codein.withgoogle.com/dashboard/task-instances/5427888430514176/)

### CODE REVIEW

**Comment by fluctix** [comment]
@logankilpatrick Please review. Not quite sure why Travis is red...

**Comment by fluctix** [comment]
My bad... Corrected it.

File: snippets/degrees_to_rads.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+title: degrees_to_rads
+tags: math,beginner
+---
+
+Converts an angle from degrees to radians.
+
+Use Julia's built in `pi` irrational data type and the degrees to radians formula to convert the angle from degrees to radians.
+
+```jl
+function degrees_to_rads(deg)
```
Comment (abhishalya) [inline snippets/degrees_to_rads.md]: I think we already have a function which does this [in Base](https://docs.julialang.org/en/v1/base/math/#Base.Math.deg2rad)

Comment (fluctix) [reply snippets/degrees_to_rads.md]: @abhishalya @logankilpatrick  I saw that now , as the task asked us to not create a snippet for a existing function i am closing this. Made a new PR: #34

**Pull request closed without merging**